http/index.php

```diff
@@ -350,7 +350,10 @@ function writeMatrix(array $glVersions, SimpleXMLElement $xml, Mesamatrix\Hints
                         </thead>
                         <tbody>
 <?php
-foreach ($xml->commits->commit as $commit) {
+$numCommits = \Mesamatrix::$config->getValue('info', 'commitlog_length', 10);
+$numCommits = min($numCommits, $xml->commits->commit->count());
+for ($i = 0; $i < $numCommits; ++$i) {
```

Comment:
You can accomplish this by casting the XML object to an array before passing it into `array_slice()`: `array_slice((array)$xml->commits->commit, 0, $numCommits);`